docs/dev-concepts/faq.md

```diff
@@ -169,16 +180,10 @@ Have other questions or ideas about group messaging? Post to the [XMTP discussio
 
 ### Which languages and environments does the XMTP SDK support?
 
-The XMTP SDK is written in TypeScript for JavaScript and TypeScript projects. It can be used with browser-based frontend frameworks like React and in Node.js.
+The XMTP SDK is [available for multiple languages](/sdks-and-tools#sdks), including JavaScript, Flutter, and Swift.
 
 Have other questions or ideas for future language or environment support? Post to the [XMTP discussion forum](https://github.com/orgs/xmtp/discussions).
 
 ### Which web3 libraries does the XMTP SDK require?
 
 The XMTP SDK currently requires you to use [ethers](https://ethers.org/) or another web3 library capable of supplying an [ethers Signer](https://docs.ethers.io/v5/api/signer/), such as [wagmi](https://wagmi.sh/).
-
-### Why does my app request a new signature for XMTP when I refresh my browser?
```

Comment:
Removed this question and combined concepts in https://github.com/xmtp/xmtp-dot-org/pull/270/files#diff-3de10b02d0ab6fa186d4ca1f14dff1c2c8520ec6055e394ae850dcb9f959ece1R102. 

I think they are different ways of asking the same question?